src/DotNetBridge/Entrypoints.cs

```diff
@@ -10,6 +10,11 @@
 
 [assembly: LoadableClass(typeof(void), typeof(DotNetBridgeEntrypoints), null, typeof(SignatureEntryPointModule), "DotNetBridgeEntrypoints")]
 
+[assembly: LoadableClass(VariableColumnTransform.Summary, typeof(VariableColumnTransform), null, typeof(SignatureLoadDataTransform),
+    "", VariableColumnTransform.LoaderSignature)]
```

Comment:
Is this really needed?

Reply:
Yes. When this is removed the unit tests (test_variable_column.py) no longer run and throw an exception (`Cannot load Model`).